.travis.yml

```diff
@@ -5,5 +5,10 @@ go:
 go_import_path: github.com/aws/amazon-ecs-agent
 
 install: make get-deps
+before_script:
+  # All the .go files in the agent, excluding vendor/ and testutils/ directories, and all *_test.go and *_mocks.go files
+  - GO_FILES=$(find ./agent -iname '*.go' -type f | grep -v /vendor/ | grep -v /testutils/ | grep -v _test\.go$ | grep -v _mocks\.go$)
+  - go get github.com/fzipp/gocyclo
```

Comment:
it's better to do this under `get-deps` imo. We already grab a bunch of tools under that target amd `install` is already invoking it. wdyt?

Reply:
Fixed.